reference/hoon-expressions/rune/buc.md

```diff
@@ -157,6 +157,59 @@ Irregular (structure mode): `[a b c]` is `$:(a b c)`.
 [p=0 q=%$]
 ```
 
+
+### $< "bucgal"
+
+`[%bsld p=spec q=spec]`: Filters a pre-existing mold to obtain a mold 
+that excludes a particular structure.
```

Comment:
"pre-existing" suggests some kind of restriction on the mold, maybe better to say "Filters a mold to obtain a new mold..."